src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/WebView.java

```diff
@@ -0,0 +1,62 @@
+package com.qaprosoft.carina.demo.mobile.gui.pages.android;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.demo.mobile.gui.pages.common.WebViewBase;
+import io.appium.java_client.AppiumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.events.EventFiringWebDriver;
+
+import java.util.Set;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = WebViewBase.class)
+public class WebView extends WebViewBase {
+
+    private AppiumDriver webViewDriver;
```

Comment:
- Please rewrite whole WebViewPage according to Project style guide.

Reply:
Do you have an example where you had to switch between contexts and use the pageObject pattern?